src/gluonts/dataset/repository/_uber.py

```diff
@@ -9,29 +8,20 @@
 
 
 def generate_uber_dataset(
-    dataset_path: Path, uber_freq: str, prediction_length: Optional[int] = None
+    dataset_path: Path, uber_freq: str, prediction_length: int
 ):
-    from gluonts.dataset.repository.datasets import default_dataset_path
-
-    uber_csv_path = default_dataset_path / "uber-raw-data-janjune-15.csv"
-    if not os.path.exists(uber_csv_path):
-        raise RuntimeError(
-            "The uber data is available at "
-            "https://github.com/fivethirtyeight/uber-tlc-foil-response/blob/"
-            "master/uber-trip-data/uber-raw-data-janjune-15.csv.zip "
-            "Please download the file and copy the files to this location: "
-            f"{uber_csv_path}"
-        )
-
     subsets = {"daily": "1D", "hourly": "1H"}
-
     assert (
         uber_freq.lower() in subsets
     ), f"invalid uber_freq='{uber_freq}'. Allowed values: {subsets.keys()}"
-
     freq_setting = subsets[uber_freq.lower()]
+
+    uber_url_path = (
+        "https://github.com/Hongqing-work/uber-tlc-foil-response/releases"

```

Comment:
Couldn't you just use the link from the `fivethirtyeight` repo?

https://github.com/fivethirtyeight/uber-tlc-foil-response/raw/63bb878b76f47f69b4527d50af57aac26dead983/uber-trip-data/uber-raw-data-janjune-15.csv.zip